lib/Default/SmrShip.class.php

```diff
@@ -153,30 +153,11 @@ public function update() {
 		$this->getPlayer()->update();
 	}
 	
-	public function checkPowerLevel($powerLevel) {
-		$this->db->query('SELECT weapon_type_id
-							FROM ship_has_weapon
-							JOIN weapon_type USING(weapon_type_id)
-							WHERE ' . $this->SQL . ' AND power_level = ' . $this->db->escapeNumber($powerLevel));
-		
-		switch ($powerLevel) {
-			case 5:
-				return $this->db->getNumRows() < 1;
-			break;
-			case 4:
-				return $this->db->getNumRows() < 2;
-			break;
-			case 3:
-				return $this->db->getNumRows() < 3;
-			break;
-			case 2:
-				return $this->db->getNumRows() < 4;
-			break;
-			case 1:
-				return true;
-			break;
-		}
-		return true;
+	/*
```

Comment:
The formatting for this docstring is slightly off. It should look like:
```
/**
 * Two asterisks on the first line (with one it's just a comment, not a docstring).
 * Text line asterisks aligned with the first asterisk on the first line.
 * One space between text-line asterisks and the text.
 */
```

Reply:
On the updated version, you got the first item there (two asterisks), but the spacing is off (asterisks aren't aligned and you have a tab instead of a space between the asterisk and text).